Added check for remote sound files (http)

lgillstrom opened the pull request:
Added check to see if src file is an url instead of a local file. There is probably a more elegant way to do this, but this change is the most minimal modification to support this.

Commit:
Added check for remote sound files (http)

Commit:
Added check for remote sound files (http)

SargoDarya committed:
Merge branch 'develop' into master

SargoDarya commented:
I like the idea of having remote sources, but this probably isn't the best way of doing it.

SargoDarya commented on audiosprite.js:
```diff
@@ -106,12 +106,12 @@ module.exports = function(files) {
-    
+
+    opts.logger.debug('Start processing', { file: src})
+    var remote = src.indexOf('http') == 0;
```
Better check here would be

`/^https?:\/\//.test(src)`

that's slightly more expensive but safer. Otherwise we might run into surprises.

SargoDarya commented on audiosprite.js:
```diff
@@ -106,12 +106,12 @@ module.exports = function(files) {
-      if (exists) {
-        let ffmpeg = spawn('ffmpeg', ['-i', path.resolve(src)]
+      if (exists || remote) {
+        let ffmpeg = spawn('ffmpeg', ['-i', remote?src:path.resolve(src)]
```
When using the ternary, use spaces. Otherwise it's hard to read.

SargoDarya committed:
Merge pull request #62 from tonistiigi/develop

Release 0.7.0 (develop => master)

okaybenji committed:
use dash flag with webm

SargoDarya committed:
Merge pull request #66 from okaybenji/dash-flag

use dash flag with webm

SargoDarya committed:
Remove --with-libogg from README.md

Remove the --with-libogg flag from the installation instructions for ffmpeg as the option does not exist anymore.

Fixes #64

Commit:
Merge remote-tracking branch 'origin/master'

Conflicts:
	audiosprite.js

Commit:
Fixing remote file handling

Commit:
updating version number
